packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -35,6 +35,9 @@ import { SupportItem, SupportItemLink, useSupportConfig } from '../../hooks';
 import { Link } from '../Link';
 
 type Props = {};
+type SupportButtonProps = {
+  title?: string;
+};
```

Comment:
shouldn't this just be one type?

```ts
type SupportButtonProps = {
  title?: string;
  children?: React.ReactNode;
};
```

and then get rid of the `PropsWithChildren` below which is nice